portal/src/views/Docs/Docs.vue

```diff
@@ -121,47 +178,183 @@ interface RouteDef {
 	description: string;
 	method: string;
 	path: string;
-	params: swag.Operation;
+	params: swag.Spec; // changed to swag.Spec because it fit our needs better than swag.Operation
 }
+
+// Copy to clipboard stuff - lets users copy the request url to their clipboard with ease
+const { isSupported, copy } = useClipboard();
 </script>
 
 <style scoped lang="scss">
+// TODO clean up these style defs in the future
+
 @import "@style/themes.scss";
 
+body {
```

Comment:
Does this style do anything since its a scoped style?

Reply:
used this for debugging _something_ but it's not needed anymore, I took it out just now